harbors/fixtures/helsinki-harbors.json

```diff
@@ -0,0 +1,2283 @@
+[
+{
+    "model": "harbors.boattypetranslation",
+    "pk": 1,
+    "fields": {
+        "language_code": "fi",
+        "name": "Jollavene",
+        "master": 1
+    }
+},
+{
+    "model": "harbors.boattypetranslation",
+    "pk": 2,
+    "fields": {
+        "language_code": "fi",
+        "name": "Sotuvene",
```

Comment:
I guess this should be `soutuvene` unless it actually is"social security number boat"... :D

Reply:
:D good catch! 👍